.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -401,7 +401,6 @@ jobs:
             until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.downgrade-vars.outputs.sha }} &> /dev/null; do sleep 45s; done
           done
 
-
```

Comment:
Nit. Should be reverted, to also remove a couple of code-owner review requests.

Reply:
done.